Assets/_Project/Scripts/Map/LocationPermissionRequester.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Mapbox.Unity.Location;
+using UnityEngine;
+using UnityEngine.Android;
+
+namespace PolSl.UrbanHealthPath.Navigation
+{
+    
+    public static class LocationPermissionRequester
```

Comment:
Please, avoid making anything static (exception being extension methods and some really basic helpers). Rewrite this as a nonstatic class with instance method.

Reply:
Done.